sites/ademe/src/backend/ademe/settings.py

```diff
@@ -0,0 +1,719 @@
+                environ_name="DB_ENGINE",
+                environ_prefix=None,
+            ),
+            "NAME": values.Value("ademe", environ_name="DB_NAME", environ_prefix=None),
+            "USER": values.Value("ademe", environ_name="DB_USER", environ_prefix=None),
+            "PASSWORD": values.Value(
+                "pass", environ_name="DB_PASSWORD", environ_prefix=None
+            ),
+            "HOST": values.Value(
+                "localhost", environ_name="DB_HOST", environ_prefix=None
+            ),
+            "PORT": values.Value(5432, environ_name="DB_PORT", environ_prefix=None),
+        }
+    }
+    MIGRATION_MODULES = {}
+
+    # Static files (CSS, JavaScript, Images)
+    STATIC_URL = "/static/"
+    MEDIA_URL = "/media/"
+    MEDIA_ROOT = os.path.join(DATA_DIR, "media")
+    STATIC_ROOT = os.path.join(DATA_DIR, "static")
+
+    # For static files, we want to use a backend that includes a hash in
+    # the filename, that is calculated from the file content, so that browsers always
+    # get the updated version of each file.
+    STATICFILES_STORAGE = values.Value("base.storage.CDNManifestStaticFilesStorage")
+
+    AUTHENTICATION_BACKENDS = ("django.contrib.auth.backends.ModelBackend",)
+
+    # AUTHENTICATION DELEGATION
+    RICHIE_AUTHENTICATION_DELEGATION = {
+        "BASE_URL": values.Value(
+            "", environ_name="AUTHENTICATION_BASE_URL", environ_prefix=None
+        ),
+        "BACKEND": values.Value(
+            "openedx-hawthorn",
+            environ_name="AUTHENTICATION_BACKEND",
+            environ_prefix=None,
+        ),
+        # PROFILE_URLS are custom links to access to Auth profile views
+        # from Richie. Link order will reflect the order of display in frontend.
+        # (i) Info - {base_url} is AUTHENTICATION_DELEGATION.BASE_URL
+        # (i) If you need to bind user data into href url, wrap the property between ()
+        # e.g: for user.username = johndoe, /u/(username) will be /u/johndoe
+        "PROFILE_URLS": values.DictValue(
+            {
+                "dashboard": {
+                    "label": _("Dashboard"),
+                    "href": _("{base_url:s}/dashboard"),
+                },
+                "profile": {
+                    "label": _("Profile"),
+                    "href": _("{base_url:s}/u/(username)"),
+                },
+                "account": {
+                    "label": _("Account"),
+                    "href": _("{base_url:s}/account/settings"),
+                },
+            },
+            environ_name="AUTHENTICATION_PROFILE_URLS",
+            environ_prefix=None,
+        ),
+    }
+
+    # LMS
+    RICHIE_LMS_BACKENDS = [
+        {
+            "BACKEND": values.Value(
+                "richie.apps.courses.lms.edx.EdXLMSBackend",
+                environ_name="EDX_BACKEND",
+                environ_prefix=None,
+            ),
+            "JS_BACKEND": values.Value(
+                "openedx-hawthorn",
```

Comment:
@jbpenrath Ah

Reply:
@sveetch so we must put 
```suggestion
                "openedx-dogwood",
```